packages/js/src/utils/generators.ts

```diff
@@ -0,0 +1,53 @@
+/** @link [Proof Key for Code Exchange by OAuth Public Clients](https://datatracker.ietf.org/doc/html/rfc7636) */
+
+import { UrlSafeBase64 } from '@silverhand/essentials';
+import { fromUint8Array } from 'js-base64';
+
+/**
+ * Length Range of Code Verifier
+ *
+ * The length of code verifier ranges from 43 to 128.
+ * @link [Client Creates a Code Verifier](https://datatracker.ietf.org/doc/html/rfc7636#section-4.1)
+ */
+export const CODE_VERIFIER_MAX_LENGTH = 128;
```

Comment:
why do we need this constant? our sdk convention specified we hard code all random string in a raw length of 64.

Reply:
Removed.